### PR TITLE
Create 'WolfenDoom: Blade of Agony' manifest

### PR DESCRIPTION
**Pull request by Alexander-Wilms**
I couldn't download the archive the WolfenDoom team publish, since ModDB doesn't provide a stable download link. GitHub limits files to 100MB, so I uploaded it on Gitlab.

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build com.realm667.WolfenDoom-Blade-of-Agony

**Comment by nedrichards** [comment]
great that we're using upstream assets. it'd be awesome if we could send a PR for OARS and versions tags as well.

**Comment by Alexander-Wilms** [comment]
I haven't completed the game yet; once I have I'll send a PR.

Should I change the dashes to underscores? The Freedoom packages' IDs contain dashes as well, and haven't caused any issues so far.

**Comment by fishxz** [comment]
the game doesn't have access to .config/gzdoom, so nothing gets stored.

**Comment by Alexander-Wilms** [comment]
You're right. It's fixed now.

**Comment by nedrichards** [comment]
bot, build com.realm667.WolfenDoom-Blade-of-Agony

**Comment by nedrichards** [comment]
@TingPing what's your thoughts on the app id question?

**Comment by Alexander-Wilms** [comment]
I've renamed it to use underscores now.

**Comment by fishxz** [comment]
i completed the game and created an ORARS. im not entirely  sure about the violence part, i would call it "Cartoon Violence". https://pastebin.com/BDGrm7Bs

**Comment by Alexander-Wilms** [comment]
Thanks. I'd add <content_attribute id="language-discrimination">intense</content_attribute>, though, due to the anti-jewish propaganda posters.

**Comment by nedrichards** [comment]
com.realm667.WolfenDoom_Blade_of_Agony

**Comment by nedrichards** [comment]
bot, build com.realm667.WolfenDoom_Blade_of_Agony

**Comment by Alexander-Wilms** [comment]
Ah, I forgot to rename the desktop and appdata file. I'll create a PR upstream adding OARS and release info and changing the ID.

**Comment by fishxz** [comment]
@Alexander-Wilms sure, feel free to use it. thats what i made it for :)

**Comment by Alexander-Wilms** [comment]
Changes have been merged upstream, and the build succeeds now.

**Comment by nedrichards** [comment]
bot, build com.realm667.WolfenDoom_Blade_of_Agony

**Comment by Alexander-Wilms** [comment]
This should be ready to merge now.

File: com.realm667.WolfenDoom_Blade_of_Agony.json [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+	"command": "gzdoom.sh",
+	"rename-icon": "m_doom",
+	"finish-args": [
+		"--device=dri",
+		"--socket=wayland",
+		"--socket=x11",
+		"--share=ipc",
+		"--socket=pulseaudio",
+		"--env=DOOMWADDIR=/app/share/games/doom",
+		"--filesystem=xdg-config/gzdoom"
```
Comment (TingPing) [inline com.realm667.WolfenDoom_Blade_of_Agony.json]: This shouldn't be here, use `--persist` when XDG isn't respected.

Comment (Alexander-Wilms) [reply com.realm667.WolfenDoom_Blade_of_Agony.json]: Done

**Comment by nedrichards** [comment]
bot, build com.realm667.WolfenDoom_Blade_of_Agony

**Comment by TingPing** [comment]
bot, build com.realm667.WolfenDoom_Blade_of_Agony

**Comment by Alexander-Wilms** [comment]
Looks like force-pushing a fix for a typo broke the builds.

**Comment by Barthalion** [comment]
bot, build com.realm667.WolfenDoom_Blade_of_Agony

**Comment by Alexander-Wilms** [comment]
I believe this is finally ready.

**Comment by TingPing** [comment]
@Alexander-Wilms You were discussing patching it to respect XDG, it would be best to do that first so you don't break configs later.

**Comment by Alexander-Wilms** [comment]
Oh, right. That'll have to wait a bit.

**Comment by Alexander-Wilms** [comment]
Just for the record, I tried patching GZDoom, but in the if(mkdir... condition, configPath somehow still contained /home/username/.config, even though it should be /home/username/.var/apps/...

[0001-Use-XDG_CONFIG_HOME-if-it-s-defined.patch.txt](https://github.com/flathub/flathub/files/1836383/0001-Use-XDG_CONFIG_HOME-if-it-s-defined.patch.txt)

**Comment by nedrichards** [comment]
bot, build com.realm667.WolfenDoom_Blade_of_Agony

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.realm667.WolfenDoom_Blade_of_Agony

**Comment by Alexander-Wilms** [comment]
Thanks!

**Pull request closed without merging**